cluster/ClusterClient.java

```diff
@@ -113,7 +114,7 @@ ClusterSession run(ClusterDatabase.Replica replica) {
         };
     }
 
-    ConcurrentMap<String, ClusterDatabase> clusterDatabases() {
+    ConcurrentMap<String, ClusterDatabase> databaseByAddress() {
```

Comment:
I love the new name, much clearer about the purpose! But also inaccurate... I believe the keys in this map are database names, not server addresses.

So we should probably rename this to `databaseByName`.

Reply:
👍